### PR TITLE
Make library items wide aspect ratio

### PR DESCRIPTION
**Pull request by danirabbit**
These days the cover art for streaming services, your legally obtained Blender open movies, etc are all wide screen format, and when cover art isn't available this format fits the most likely format of your video:

Screenshot from 2024-05-22 11 31 20

### CODE REVIEW

File: src/Services/LibraryManager.vala [hunk 1]
```diff
@@ -20,8 +20,8 @@
  */
 
 namespace Audience.Services {
-    public const int POSTER_WIDTH = 170;
-    public const int POSTER_HEIGHT = 240;
+    public const int POSTER_WIDTH = 300;
+    public const int POSTER_HEIGHT = (int) (300 * 9 / 16);
```
Comment (zeebok) [inline src/Services/LibraryManager.vala:24]: Why the full math instead of just the value?

Comment (danirabbit) [reply src/Services/LibraryManager.vala:24]: So that it’s clear what this number is and where it comes from and it isn’t just a magic number

Comment (davidmhewitt) [reply src/Services/LibraryManager.vala:24]: Should POSTER_HEIGHT use POSTER_WIDTH instead of writing 300 again?

Comment (danirabbit) [reply src/Services/LibraryManager.vala]: Yes, done!

**Review by davidmhewitt — APPROVED** [review]

**Pull request merged**